qiskit_nature/second_q/drivers/psi4d/__init__.py

```diff
@@ -0,0 +1,34 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
```

Comment:
The original `Psi4` driver files all still contain 2018, too. Do we want to retain that year since the code is basically still the same? (same applies to other files)